### DISCUSSION TITLE
[Item] my Template

### DISCUSSION THREAD

**Discussion by Felix-fz** [windingwind/zotero-better-notes#376] [Note Templates]
下面是我的解决方案及效果图，大家喜欢的话可以进行交流，后续也会继续创作一些好看的模板。开源链接：https://github.com/CopleM/ZoteroNotesTemplate/

![image](https://user-images.githubusercontent.com/61495551/221558436-dd1f7800-a2cc-4548-8c53-219e7ff58abf.png)

**Comment by EdwardGiao** [comment]
大佬怎么把第一行的标题改为作者+发表年呐？
